src/current/_includes/common/set-enterprise-license.md

```diff
@@ -0,0 +1,11 @@
+Start the [`cockroach sql` shell]({% link {{ page.version.version }}/cockroach-sql.md %}). As the [`root` user]({% link {{ page.version.version }}/security-reference/authorization.md %}#root-user), use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement to set the name of your organization and the license key:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SET CLUSTER SETTING cluster.organization = 'Acme Company';
```

Comment:
The cluster.organization setting is no longer required. We used to verify that the organization in the license matched this setting, but that's no longer the case. Users can now leave it blank. Are we keeping this setting around to support users who might be following updated documentation on older servers?

Reply:
> The cluster.organization setting is no longer required. We used to verify that the organization in the license matched this setting, but that's no longer the case. Users can now leave it blank.

Awesome, in that case I will remove it

> Are we keeping this setting around to support users who might be following updated documentation on older servers?

i didn't have such an intention, it was due to ignorance on my part :-) 

given your other comment that it is no longer required i will remove it